### PR TITLE
[full-ci] [tests-only] Cleanup spaces created by test scenario after execution of tests

### PR DESCRIPTION
**Pull request by amrita-shrestha**
## Description
This issue clean-up the spaces after the test scenario run. 
## Related Issue
- https://github.com/owncloud/web/issues/8662

## How Has This Been Tested?

- locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/34639/12/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/34639/tracing/spaces-can-be-managed-in-the-admin-settings-via-the-context-menu-alice-2023-4-12-04-28-59.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/34639/tracing/spaces-can-be-managed-in-the-admin-settings-via-the-context-menu-brian-2023-4-12-04-29-06.zip 


</p></details>

**Comment by SwikritiT** [comment]
@amrita-shrestha can we also do this https://github.com/owncloud/web/pull/8661#discussion_r1144532986 in this same PR so that we know this fix is working maybe?

File: tests/e2e/support/api/http.ts [hunk 1]
```diff
@@ -9,25 +9,31 @@ export const request = async ({
   path,
   body,
   user,
-  formatJson = true
+  formatJson = true,
+  header = null
```
Comment (saw-jan) [inline tests/e2e/support/api/http.ts:13]:
```suggestion
  header = {}
```

File: tests/e2e/support/api/http.ts [hunk 2]
```diff
@@ -9,25 +9,31 @@ export const request = async ({
...
   const format = config.ocis || !formatJson ? '' : 'format=json'
+  let basicHeader = {
+    'OCS-APIREQUEST': true as any,
+    ...(user && {
+      Authorization: 'Basic ' + Buffer.from(user.id + ':' + user.password).toString('base64')
+    })
+  }
+  if (header) {
+    basicHeader = Object.assign(basicHeader, header)
+  }
```
Comment (saw-jan) [inline tests/e2e/support/api/http.ts:31]:
```suggestion
  const basicHeader = {
    'OCS-APIREQUEST': true as any,
    ...(user && {
      Authorization: 'Basic ' + Buffer.from(user.id + ':' + user.password).toString('base64')
    }),
    ...header
  }
```

**Review by saw-jan — COMMENTED** [review]
Maybe can be done like this

**Review by saw-jan — COMMENTED** [review]
LGTM

**Review by SwikritiT — COMMENTED** [review]
LGTM. It says that the branch is 32 commits behind the master so please make you you rebase before merging

**Pull request merged**